Source/Core/Core/State.cpp

```diff
@@ -192,8 +192,8 @@ static void DoState(PointerWrap& p)
 
   // Movie must be done before the video backend, because the window is redrawn in the video backend
   // state load, and the frame number must be up-to-date.
-  Movie::DoState(p);
-  p.DoMarker("Movie");
+  // Movie::DoState(p);
+  // p.DoMarker("Movie");
```

Comment:
Why was this removed? (Not that I'm sure if TASing is a supported use-case for Slippi...)